### PR TITLE
LPS-183744 Add Account Group On Site Initializer

### PR DESCRIPTION
**Pull request by joaodamasceno2001**

### CODE REVIEW

**Comment by joaodamasceno2001** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-1/src/main/resources/site-initializer/account-groups.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[
+	{
+		"externalReferenceCode": "ERC-01",
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-1/src/main/resources/site-initializer/account-groups.json:3]: please rename ERC to TESTACCOUNTGROUP1 ...

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 2]
```diff
@@ -222,6 +224,7 @@ public class BundleSiteInitializer implements SiteInitializer {
 
 	public BundleSiteInitializer(
 		AccountResource.Factory accountResourceFactory,
+		AccountGroupResource.Factory accountGroupResourceFactory,
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:227]: sort

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 3]
```diff
@@ -293,6 +296,7 @@ public BundleSiteInitializer(
 		WorkflowDefinitionResource.Factory workflowDefinitionResourceFactory) {
 
 		_accountResourceFactory = accountResourceFactory;
+		_accountGroupResourceFactory = accountGroupResourceFactory;
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:299]: sort

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java [hunk 4]
```diff
@@ -2668,6 +2732,7 @@ private void _test1(SiteInitializer siteInitializer) throws Exception {
 		siteInitializer.initialize(_group.getGroupId());
 
 		_assertAccounts1();
+		_assertAccountGroups1();
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java:2735]: sort

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java [hunk 5]
```diff
@@ -2710,6 +2775,7 @@ private void _test2(SiteInitializer siteInitializer) throws Exception {
 		siteInitializer.initialize(_group.getGroupId());
 
 		_assertAccounts2();
+		_assertAccountGroups2();
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java:2778]: sort

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 6]
```diff
@@ -447,6 +451,8 @@ public void initialize(long groupId) throws InitializationException {
 
 			_invoke(() -> _addAccounts(serviceContext));
 
+			_invoke(() -> _addAccountGroups(serviceContext));
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:454]: sort

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/SiteInitializerExtension.java [hunk 7]
```diff
@@ -171,9 +173,9 @@ public SiteInitializerExtension(
 		_component = _dependencyManager.createComponent();
 
 		BundleSiteInitializer bundleSiteInitializer = new BundleSiteInitializer(
-			accountResourceFactory, accountRoleLocalService,
-			accountRoleResourceFactory, assetCategoryLocalService,
-			assetListEntryLocalService, bundle,
+			accountResourceFactory, accountGroupResourceFactory,
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/SiteInitializerExtension.java:176]: sort

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/SiteInitializerFactoryImpl.java [hunk 8]
```diff
@@ -119,9 +120,9 @@ public SiteInitializer create(File file, String symbolicName)
 			null);
 
 		BundleSiteInitializer bundleSiteInitializer = new BundleSiteInitializer(
-			_accountResourceFactory, _accountRoleLocalService,
-			_accountRoleResourceFactory, _assetCategoryLocalService,
-			_assetListEntryLocalService, bundle,
+			_accountResourceFactory, _accountGroupResourceFactory,
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/SiteInitializerFactoryImpl.java:123]: sort

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-1/src/main/resources/site-initializer/account-groups.json [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+		"name": "Test 1"
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-1/src/main/resources/site-initializer/account-groups.json:4]: please rename to Test Account Group 1

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Review by nNilton — APPROVED** [review]

**Comment by nNilton** [comment]
waiting for @joseabelenda review

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by nNilton** [comment]
ci:test:relevant

**Pull request closed without merging**